contracts/utils/GlobalRateLimitedMinter.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "./MultiRateLimited.sol";
+import "./IGlobalRateLimitedMinter.sol";
+
+/// @notice global contract to handle rate limited minting of FEI on a global level
+/// allows whitelisted minters to call in and specify the address to mint FEI to within
+/// that contract's limits
+contract GlobalRateLimitedMinter is MultiRateLimited, IGlobalRateLimitedMinter {
+    /// @param coreAddress address of the core contract
+    /// @param _globalMaxRateLimitPerSecond maximum amount of fei that can replenish per second
+    /// @param _globalRateLimitPerSecond maximum global buffer cap
+    /// @param _bufferCap maximum rate limit per second per minter
+    /// @param maxBufferCap maximum buffer cap per minter
+    /// @param _doPartialAction boolean to describe whether partial actions are allowed or not
+    constructor(
+        address coreAddress,
+        uint256 _globalMaxRateLimitPerSecond,
+        uint256 _globalRateLimitPerSecond,
```

Comment:
I'm not sure if these argument names (`_globalMaxRateLimitPerSecond`,  `_globalRateLimitPerSecond`) are the clearest or how they map up to the description in the natspec comments. 

`_globalRateLimitPerSecond maximum global buffer cap` seems to suggest this is a maximum global constant, rather than a rate? Would `_globalMaxBufferCap` be more accurate?

Reply:
I've updated the comments in the constructor args description. Please check them and let me know what you think.